src/olympia/editors/views.py

```diff
@@ -482,12 +482,24 @@ def construct_query(query_type, days_min=None, days_max=None):
 
         return query.count
 
+    AUTO_APPROVED = amo.AUTO_APPROVED
+    a_year_ago = datetime.now() - timedelta(days=365)
```

Comment:
?
Is `a year` a predefined period of time?  Seems like this should be a constant/setting.

Reply:
It's arbitrary, but I don't think it warrants a setting. I've moved the queryset to its own method to keep things DRY, if we need to change it we'll just need to change this new method.